MeetupPlatformApi/Authentication/TokenPair.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace MeetupPlatformApi.Authentication;
+
+using MeetupPlatformApi.Entities;
+
+public class TokenPair
+{
+    public string AccessToken { get; set; }
+
+    public RefreshTokenEntity RefreshToken { get; set; }
```

Comment:
This model should contain already encoded token.
We did move all stuff related token issuing, validation and parsing to a separate manager to have an isolation between application logic and token-related logic. Please, do not break this isolation by directly referencing entities.